examples/WebClient/WebClient.ino

```diff
@@ -0,0 +1,167 @@
+/*
+  Web client
+
+  This sketch connects to a website (http://www.google.com)
+  using an Arduino Wiznet Ethernet shield.
+
+  Circuit:
+   Ethernet shield attached to pins 10, 11, 12, 13
+
+  created 18 Dec 2009
+  by David A. Mellis
+  modified 9 Apr 2012
+  by Tom Igoe, based on work by Adrian McEwen
+
+*/
+
+#include <QNEthernet.h>
+
+using namespace qindesign::network;
+
+// if you don't want to use DNS (and reduce your sketch size)
+// use the numeric IP instead of the name for the server:
+char server[] = "arduino.cc";  // also change the Host line in httpRequest()
+//IPAddress server(100,24,172,113);
+
+#define USING_DHCP    true
+
+#if !USING_DHCP
+  // Set the static IP address to use if the DHCP fails to assign
+  IPAddress myIP(192, 168, 2, 222);
+  IPAddress myNetmask(255, 255, 255, 0);
+  IPAddress myGW(192, 168, 2, 1);
+  IPAddress mydnsServer(8, 8, 8, 8);
+#endif
+
+// Initialize the Ethernet client library
+// with the IP address and port of the server
+// that you want to connect to (port 80 is default for HTTP):
+EthernetClient client;
+
+// Variables to measure the speed
+unsigned long beginMicros, endMicros;
+unsigned long byteCount = 0;
+bool printWebData = true;  // set to false for better speed measurement
+
+void setup()
+{
+  // Open serial communications and wait for port to open:
+  Serial.begin(115200);
+  while (!Serial);
+
+  Serial.println("\nStarting WebClient with NativeEthernet using QNEthernet Library");
+
+  // start the Ethernet connection:
+  Serial.println("Initialize Ethernet with DHCP:");
```

Comment:
Move comment to both DHCP and static IP sections because "DHCP" shouldn't be printed for both cases.

Reply:
Will do like this

```
#if USING_DHCP
  // Start the Ethernet connection, using DHCP
  Serial.print("Initialize Ethernet using DHCP => ");
  Ethernet.begin();
#else  Ethernet.setDNSServerIP(mydnsServer);
  
  // Start the Ethernet connection, using static IP
  Serial.print("Initialize Ethernet using static IP => ");
  Ethernet.begin(myIP, myNetmask, myGW);
#endif
```